common/pref_names.h

```diff
@@ -43,9 +43,9 @@ extern const char kShieldsAdvancedViewEnabled[];
 extern const char kShieldsStatsBadgeVisible[];
 extern const char kAdControlType[];
 extern const char kGoogleLoginControlType[];
-extern const char kFBEmbedControlType[];
-extern const char kTwitterEmbedControlType[];
-extern const char kLinkedInEmbedControlType[];
+// extern const char kFBEmbedControlType[];
+// extern const char kTwitterEmbedControlType[];
+// extern const char kLinkedInEmbedControlType[];
```

Comment:
Same here, looks like you can remove this commented lines (also in components/brave_shields/common/pref_names.h)